tests/sentry/integrations/vsts/test_issues.py

```diff
@@ -212,6 +218,60 @@ def test_get_issue_url(self):
         url = self.integration.get_issue_url(work_id)
         assert url == 'https://fabrikam-fiber-inc.visualstudio.com/_workitems/edit/345'
 
+    @responses.activate
+    def test_should_resolve_active_to_resolved(self):
+        should_resolve = self.integration.should_resolve({
+            'project': self.project_id_with_states,
+            'old_state': 'Active',
+            'new_state': 'Resolved',
+        })
+        assert should_resolve is True
+
+    @responses.activate
+    def test_should_resolve_resolved_to_active(self):
+        should_resolve = self.integration.should_resolve({
+            'project': self.project_id_with_states,
+            'old_state': 'Resolved',
+            'new_state': 'Active',
+        })
+        assert should_resolve is False
+
+    @responses.activate
+    def test_should_resolve_new(self):
+        should_resolve = self.integration.should_resolve({
+            'project': self.project_id_with_states,
+            'old_state': None,
+            'new_state': 'New',
+        })
+        assert should_resolve is False
```

Comment:
Could we get a hook that has `old_state = None`, `new_state = 'Resolved'`?

Reply:
That sounds like a bug honestly on VSTS's end. That's a bit strange to me. :O .... I've never seen issues in VSTS begin as resolved. And I only see this happening when the issue is new. I'm not sure we _should_ handle that case if it exists... more like log the error. The docs aren't really reliable, so I can only say by what I've seen so far.